quiz.js

```diff
@@ -1,5 +1,4 @@
-const questions = [
-  {
+const questions = [{
     question: "Nazwa zmiennej nie może zaczynać się od:",
```

Comment:
Masz inne formatowanie w edytorze, które przenosi do jednej linii to, co było w dwóch. Tym się na razie nie przejmujemy, chociaż w niektórych repozytoriach się mogą czepiać więc informuję :)